lib/ahoy_activity_utils.rb

```diff
@@ -0,0 +1,67 @@
+module AhoyActivityUtils
+    def self.rollup_transcribe_for_date(day=1.day.ago)
+        # Active users are users who log "transcription-type" deeds on a given day.
+        # We only want to track these people for now
+        active_users = Deed
+        .where(deed_type: DeedType.transcriptions)
+        .where("created_at BETWEEN ? AND ?", day.beginning_of_day, day.end_of_day)
+        .distinct.pluck(:user_id)
```

Comment:
Is there a chance this `distinct` is what's slowing us down?  At times, testing the uniqueness in SQL is really slow if there is no index (in which case it's blazingly fast).